### ISSUE TITLE
Image does not contain a text editor

### ISSUE THREAD

**Issue by clach04** [bboehmke/docker-pebble-dev#7]
Related to issue #3, its not possible to edit nor interactive commit using git (`git -m` helps with the later).

Any objections to adding a text editor to the image? I'd prefer it to be vim but I understand if something smaller and light weight (line nano, pico, ne, etc.) is preferred.

Hitting this due to issue #6 under Windows host - and this seems the least path of resistance to a working solution.

**Comment by bboehmke** [comment]
Based on the current image size of 750 MB I dont think that it would be a problem to add vim.

An other option can be to add an optional SSH server to the image so you can access the files from outside or even use something like [sshcode](https://github.com/cdr/sshcode).

Feel free to create a pull request.

**Comment by clach04** [comment]
> An other option can be to add an optional SSH server to the image so you can access the files from outside or even use something like [sshcode](https://github.com/cdr/sshcode).

Interesting idea. I'm still a novice with Docker so I went with a quick PR for vim and screen addition. I did them as separate apt-get from the existing required packages so that they can be easily removed (or add additional optional packages).

I read some articles that indicted it was preferable to avoid sshd, this maybe one of those unique cases where its a good idea. But you got me thinking about installed code-server (or sshd/ftp to allow external editor support). I skipped installing sshd by default so that there are no unexpected behaviors for anyone pulling this down who doesn't need (or want) it listening.
